src/frontend/src/helpers/Login.ts

```diff
@@ -0,0 +1,59 @@
+import * as $ from "jquery";
+export {Login, LoginError, Connection}
+
+class LoginError extends Error {
+  constructor(msg: string,
+              public username?: string,
+              public status?: number,
+              public statusText?: number) {
+    super(msg);
+  }
+}
+
+class Login {
```

Comment:
Do you mean I should put it in a namespace or make a static class? Or just a global function
Maybe the class name is poorly named. There should also be a `logout` method in there.